### PR TITLE
fix(node-builtins): Remove "node:" prefix from "ignores" message

### LINKED ISSUE #268: New option: Support `"fetch"` in `"ignore"` option of `n/no-unsupported-features/node-builtins`
### Rule details

This issue doesn't propose a new rule, but a new value for the `"ignore"` option of the [`n/no-unsupported-features/node-builtins`](https://github.com/eslint-community/eslint-plugin-n/blob/78595c42642349c5f5bb2e0ae9986da9ae93fed8/docs/rules/no-unsupported-features/node-builtins.md#ignores) rule.

With Node v20, this rule reports that `fetch` is experimental. I'd like to disable `fetch` checking by adding it to the list of ignored features.

### What type of rule is this?

Suggests an alternate way of doing something

### Example code

```js
export default {
  "rules": {
    "n/no-unsupported-features/es-builtins": [
      "error",
      { ignores: ["fetch"] }
    ]
  }
};
```

### Participation

- [ ] I am willing to submit a pull request to implement this rule.

### Additional comments

_No response_

### PR DESCRIPTION
**Pull request by scagood**
fixes #268

While updating the docs for #268 I noticed that we're passing the raw path through to the name!

I have removed that at the same time.

### CODE REVIEW

File: docs/rules/no-unsupported-features/node-builtins.md [hunk 1]
```diff
@@ -49,273 +49,16 @@ The `"allowExperimental"` option accepts a boolean value (the default value is `
...
-- `"worker_threads"`
-
-</details>
+The `"ignores"` option accepts an array of strings.
+
+> [!TIP]
+> You will see the following error messages:
+>
+> - `The '{{name}}' is not an experimental feature until Node.js {{version}}.`
+> - `The '{{name}}' is still an experimental feature and is not supported until Node.js {{version}}.`gi
```
Comment (aladdin-add) [inline docs/rules/no-unsupported-features/node-builtins.md:58]: is the `gi` intentional?

File: docs/rules/no-unsupported-features/node-builtins.md [hunk 2]
```diff
@@ -49,273 +49,16 @@ The `"allowExperimental"` option accepts a boolean value (the default value is `
...
+> - `The '{{name}}' is still an experimental feature`
+>
+> The "name" property is what you can use in your `ignores` array
```
Comment (aladdin-add) [inline docs/rules/no-unsupported-features/node-builtins.md:61]: maybe adding a link to the modules: https://github.com/eslint-community/eslint-plugin-n/tree/HEAD/lib/unsupported-features/node-builtins-modules?

File: docs/rules/no-unsupported-features/node-builtins.md (hunk 1 above)
Comment (scagood) [reply docs/rules/no-unsupported-features/node-builtins.md:58]: I was wondering where the start of my `git commit` went 😱 Good spot!

```suggestion
> - `The '{{name}}' is still an experimental feature and is not supported until Node.js {{version}}.`
```

File: docs/rules/no-unsupported-features/node-builtins.md (hunk 2 above)
Comment (scagood) [reply docs/rules/no-unsupported-features/node-builtins.md:61]: I like that idea.
```suggestion
> The "name" property is what you can use in your `ignores` array
> You can find the full tree view of the list of the modules over in [lib/unsupported-features/node-builtins-modules](https://github.com/eslint-community/eslint-plugin-n/tree/HEAD/lib/unsupported-features/node-builtins-modules). and globals over in [lib/unsupported-features/node-globals.js](https://github.com/eslint-community/eslint-plugin-n/blob/HEAD/lib/unsupported-features/node-globals.js)
```

**Review by aladdin-add — APPROVED** [review]

**Pull request merged**